server/api.go

```diff
@@ -211,21 +205,6 @@ func (p *Plugin) completeConnectUserToGitHub(w http.ResponseWriter, r *http.Requ
 	w.Write([]byte(html))
 }
 
-func (p *Plugin) handleProfileImage(w http.ResponseWriter, r *http.Request) {
```

Comment:
Removing this will break existing posts. 3/5 that we should not do this. At least until `v1.0`.

Reply:
It won't because it was not actually used. See: https://github.com/mattermost/mattermost-plugin-github/pull/83/files#diff-97fcb5163c5df39ace07e8d58616214eL108